### ISSUE TITLE
Windows Libtorch Ver1.5 did't operate at the GPU mode , it operated at the CPU mode. 

### ISSUE THREAD

**Issue by yoshihingis** [pytorch/pytorch#37568]
## 🐛 Bug
Windows Libtorch Ver1.5 did not operate at the GPU mode , it operated at the CPU mode.

##Enviroment
・OS : Windows 10 Pro 64bit
・Compiler: Visual studio 2019 Professional
・Test code: mnist.cpp (got from github)
・libtorch Ver1.5 : I dounloaded  https://download.pytorch.org/libtorch/cu102/libtorch-win-shared-with-deps-1.5.0.zip
   Stable(1.5),, Windows, Libtorch, C++/Java ,CUDA10.2 ,Release mode
・CUDA10.2
・cuDNN 7.6.532 for CUDA10.2
・GPU : Geforce GTX1060 (6GB memory) ZOTAC

## To Reproduce
1 .I donwloaded libtorch-win-shared-with-deps-1.5.0.zip from https://pytorch.org.

2. I unziped  libtorch-win-shared-with-deps-1.5.0.zip.

3. I got mnist.cpp form https://github.com/pytorch/examples/tree/master/cpp/mnist
[mnist.txt](https://github.com/pytorch/pytorch/files/4556126/mnist.txt)

4. I made  New Project "mnistv1_5_0430" by Visual studo 2019
                                                                                                                                                                                                                                                          
5. I did the setting of source code, include path , add user lib files,and etc.                                                                                                                                       

6.  I build the project and the project was built with no erros. 

7. I get executable file "mnistv1_5_0430.exe"                                                                                                                                                                                                                                                                                                                                                           
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
8. I downloaded data for learning by download_mnist.py at folder " ./data".
                                                                                                                                                                                                                                                                                                                                              
9.I opened "Command promt" as Administrator .

10. At first I run nvidia-smi.exe  ,the result was as blow.
    I confirmed that I installed Geforce GTX1060 and CUDA10.2.                                                                                                                                                                                          
                                                                                                                                                                        
 C:\Program Files\NVIDIA Corporation\NVSMI>nvidia-smi.exe
Thu Apr 30 02:27:21 2020
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 441.22       Driver Version: 441.22       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name            TCC/WDDM | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 106... WDDM  | 00000000:01:00.0  On |                  N/A |
| 40%   31C    P8     7W / 120W |    293MiB /  6144MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0       788    C+G   ...1.0_x64__8wekyb3d8bbwe\WinStore.App.exe N/A      |
|    0      1256    C+G   Insufficient Permissions                   N/A      |
|    0      6700    C+G   ...R.x86\ServiceHub.ThreadedWaitDialog.exe N/A      |
|    0      8104    C+G   C:\Windows\explorer.exe                    N/A      |
|    0      8736    C+G   ...5n1h2txyewy\StartMenuExperienceHost.exe N/A      |
|    0      9028    C+G   ...dows.Cortana_cw5n1h2txyewy\SearchUI.exe N/A      |
|    0      9436    C+G   ....111.0_x64__8wekyb3d8bbwe\YourPhone.exe N/A      |
|    0      9932    C+G   ...6.102.0_x64__kzf8qxf38zg5c\SkypeApp.exe N/A      |
|    0     10080    C+G   ...DIA GeForce Experience\NVIDIA Share.exe N/A      |
|    0     10476    C+G   ...t_cw5n1h2txyewy\ShellExperienceHost.exe N/A      |
|    0     10812    C+G   ...hell.Experiences.TextInput.InputApp.exe N/A      |
|    0     11160    C+G   ...019\Professional\Common7\IDE\devenv.exe N/A      |
|    0     12552    C+G   ...mmersiveControlPanel\SystemSettings.exe N/A      |
+-----------------------------------------------------------------------------+
                                                  
 11. I did "mnistv1_5_0430.exe" and the learning of mnist started as blow
       Please look first message "Training on CPU.", GPU was not used.
 
D:\Konicaminolta\0430\mnistv1_5_0430\x64\Release>mnistv1_5_0430.exe

Training on CPU.
Train Epoch: 1 [59584/60000] Loss: 0.2362
Test set: Average loss: 0.2087 | Accuracy: 0.933
Train Epoch: 2 [59584/60000] Loss: 0.1832
Test set: Average loss: 0.1284 | Accuracy: 0.960
Train Epoch: 3 [59584/60000] Loss: 0.1072
Test set: Average loss: 0.1010 | Accuracy: 0.970
Train Epoch: 4 [59584/60000] Loss: 0.1157
Test set: Average loss: 0.0852 | Accuracy: 0.973
Train Epoch: 5 [59584/60000] Loss: 0.0822
Test set: Average loss: 0.0783 | Accuracy: 0.974
Train Epoch: 6 [59584/60000] Loss: 0.0605
Test set: Average loss: 0.0695 | Accuracy: 0.978
Train Epoch: 7 [59584/60000] Loss: 0.1973
Test set: Average loss: 0.0617 | Accuracy: 0.982
Train Epoch: 8 [59584/60000] Loss: 0.0872
Test set: Average loss: 0.0595 | Accuracy: 0.982
Train Epoch: 9 [59584/60000] Loss: 0.0636
Test set: Average loss: 0.0536 | Accuracy: 0.984
Train Epoch: 10 [59584/60000] Loss: 0.0995
Test set: Average loss: 0.0505 | Accuracy: 0.984

12. But libtroch Ver1.5 operated correctly by using mnist.cpp in Ubuntu18.04LTS.
     I changed the OS Windows10→Ubuntu18.04LTS in same PC.
    We can see the message "CUDA available ! Training on GPU"
![mnist_ubuntu](https://user-images.githubusercontent.com/18341725/80673165-90d34780-8ae9-11ea-9d23-3503399fbe9c.jpg)

[BUG]
・The libtorch ver1.5 operated did not operate correctly in Windows10 ,but also it operated correctly in Ubuntu18.04LTS.

**Comment by peterjc123** [comment]
https://github.com/pytorch/pytorch/issues/37124
